src/django/pfb_analysis/views.py

```diff
@@ -70,6 +70,7 @@ def results(self, request, pk=None):
 class NeighborhoodMixin(APIView):
     """Shared properties of the neighborhood viewsets."""
 
+    queryset = Neighborhood.objects.all()
```

Comment:
Tried putting this property on the mixin when adding it initially, but it would error. Thought this simply wouldn't generalize because `ModelViewSet` and `ReadOnlyModelViewSet` separately inherit from `GenericAPIViewSet` and not from each other, but curiously, this does seem to work now.